src/GovUk.Education.ExploreEducationStatistics.Common/Model/TimeIdentifier.cs

```diff
@@ -140,6 +140,9 @@ public enum TimeIdentifier
 
         [TimeIdentifierMeta("Tax Year Q4", "TYQ4", Category.TaxYear, AcademicOrFiscalShort, "Q4")]
         TaxYearQ4,
+        
+        [TimeIdentifierMeta("Reporting Year", "RY", Category.ReportingYear, NoLabel)]
+        ReportingYear,
```

Comment:
Think there are some tests for `TimeIdentifiers` that probably need this added to

Reply:
yep done - thanks

Reply:
Tested in association with frontend and appears to work OK.